locator/ec2_snitch.cc

```diff
@@ -64,6 +65,24 @@ future<> ec2_snitch::start() {
 }
 
 future<sstring> ec2_snitch::aws_api_call(sstring addr, uint16_t port, sstring cmd) {
+    for (int i = 0; i < AWS_API_CALL_RETRIES; i++) {
+        try {
+            auto s = aws_api_call_once(addr, port, cmd).get();
```

Comment:
It's not valid to use `.get()` here, the method happens to be in seastar::async() by chance, not intentionally. Please, use `repeat_until_value`

Reply:
@xemul replaced `for()` to `repeat_until_value()`, `aws_api_call_once().get()` to `aws_api_call_once().then([] () { ... }).handle_exception([] () { ... })`, and `sleep().get()` to `sleep().then([] () { ... })`. 
Please review it again.